src/main/java/com/redhat/devtools/intellij/quarkus/telemetry/TelemetryService.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *
+ * Contributors:
+ *     Red Hat Inc. - initial API and implementation
+ *******************************************************************************/
+package com.redhat.devtools.intellij.quarkus.telemetry;
+
+import com.redhat.devtools.intellij.telemetry.core.service.TelemetryMessageBuilder;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+/**
+ * Telemetry Service facade
+ */
+public interface TelemetryService {
+
+    String LSP_PREFIX = "lsp-";
+
+    String UI_PREFIX = "ui-";
+
+    String MODEL_PREFIX = "model-";
+
+    String RUN_PREFIX = "run-";
```

Comment:
I'd say those belong to TelemetryEventName

Reply:
Indeed I wanted to do that, but it seems that it is not possible to declare constants in Java enum?

Reply:
yes you can, just make sure they're after the enum values. Also you should use 'static' as they're constants

Reply:
fixed